### DISCUSSION TITLE
missing volume name for nextcloud-aio-talk

### DISCUSSION THREAD

**Discussion by hrolofs** [nextcloud/all-in-one#3004] [Questions] [answered]
Hi,

After successfully installing Nextcloud-AIO, I am looking for what is installed on Docker.

the installed volumes are

docker volume ls
DRIVER    VOLUME NAME
local     f152ddc56dcc6d061345b20fee19a87c82b86d07e2cafa0c797b245777643b4f
local     nextcloud_aio_apache
local     nextcloud_aio_clamav
local     nextcloud_aio_database
local     nextcloud_aio_database_dump
local     nextcloud_aio_elasticsearch
local     nextcloud_aio_mastercontainer
local     nextcloud_aio_nextcloud
local     nextcloud_aio_redis

The first one are the volume that used by nextcloud-aio-talk container.

Shouldn't it be called  nextcloud_aio_talk then like the others?

Regards

**Comment by Zoey2936** [comment, marked as answer]
no, since it is a temorary volume
